iDiff/utils/tar_utils.go

```diff
@@ -32,7 +32,7 @@ func ImageToDir(img string) (string, string, error) {
 	os.Remove(tarPath)
 	path := strings.TrimSuffix(tarPath, filepath.Ext(tarPath))
 	jsonPath := path + ".json"
-	err = DirToJSON(path, jsonPath)
+	err = DirToJSON(path, jsonPath, false)
```

Comment:
Do we always want this to be false?  Would it be useful to add the boolean as an argument for ImageToDir so for a specific differ we can control the shallow or deep output?